docs/gno-tooling/cli/gnokey/full-security-tx.md

```diff
@@ -0,0 +1,128 @@
+---
+id: full-security-tx
+---
+
+# Making an airgapped transaction
+
+## Prerequisites
+
+- **`gnokey` installed.** Reference the
+  [Local Setup](../../../getting-started/local-setup/installation.md#2-installing-the-required-tools) guide for steps
+
+## Overview
+
+`gnokey` provides a way to create a transaction, sign it, and later
+broadcast it to a chain in the most secure fashion. This approach, while more 
+complicated, grants full control and provides airgap support.
+
+The indented purpose of this functionality is to provide maximum security when 
+signing and broadcasting a transaction. In practice, this procedure should take
+place on two separate machines, one with access to the internet (`Machine 1`), 
+and the other one without (`Machine 2`), with the separation of steps as follows:
+1. `Machine 1`: Fetch account information from the chain
+2. `Machine 2`: Create an unsigned transaction locally
+3. `Machine 2`: Sign the transaction
+4. `Machine 1`: Broadcast the transaction
+
+For the sake of simplicity, in this example, we will assume that the procedure 
+is happening on two machines, and we will again use the Userbook 
+realm on the Portal Loop testnet.
+
+## Fetching account information from the chain
```

Comment:
Maybe numbering the headings like the numbered list could be helpful? You could even specify `Step 1: Fetching account information from the chain.` I'm thinking again about people who jump straight to section 3 via a link it might be useful for them to understand at a glance that they’re in the middle of a sequence of steps.

Reply:
You're right, added numbers: [9c833ed](https://github.com/gnolang/gno/pull/2226/commits/9c833ede96ba006c51c2a33df05690c0d1fc76a8)